.github/workflows/cd-app-nextjs-storybook.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy Storybook
```

Comment:
Not sure how/if this is going to work in a multi-app repo as-is. But I guess we'll find out.